db/migrate/20200608175130_add_banner_style_to_broadcasts.rb

```diff
@@ -0,0 +1,5 @@
+class AddBannerStyleToBroadcasts < ActiveRecord::Migration[6.0]
+  def change
+    add_column :broadcasts, :banner_style, :string
```

Comment:
should this have a `null: false` plus default value or it doesn't matter if existing broadcasts don't have a style set?

Reply:
No, I don't think it needs this; welcome broadcasts don't require a style at all (we don't use them), and you don't even _need_ a style set for announcement broadcasts, either (though we ought to recommend it). So, there's no reason to enforce this on a broadcast-wide basis.

Reply:
ah, make sense! :)